src/main/kotlin/com/devfloor/untitled/articleviewcount/domain/ArticleViewCount.kt

```diff
@@ -0,0 +1,30 @@
+package com.devfloor.untitled.articleviewcount.domain
+
+import com.devfloor.untitled.article.domain.Article
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToOne
+
+@Entity
+class ArticleViewCount(
+    article: Article,
+) {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "article_view_count_id")
+    val id: Long = 0
+
+    @OneToOne
+    @JoinColumn(name = "article_id")
+    val article: Article = article
+
+    @Column(name = "viewCount")
+    var viewCount: Long = 1
+        protected set
```

Comment:
```suggestion
    @Column(name = "count")
    var count: Long = 1
        protected set
```

간결하게 count로 가져가는 것도 괜찮을 것 같은데 어때?

Reply:
오키~